### PR TITLE
update functionality for Admin profile

### PR DESCRIPTION
**Pull request by SwayamRana808**
## Issue that this pull request solves

 Closes: #896 

## Proposed changes
[Frontend] Introduce Update Admin Functionality
### Brief description of what is fixed or changed
  Introduce Update Admin Functionality and also backend api updated 
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

https://github.com/HITK-TECH-Community/Community-Website/assets/122097724/7b260dd2-71cb-4d47-ac23-f09d0f19de72

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: backend/app/routes/admin/getAdmins.js [hunk 1]
```diff
@@ -46,5 +47,5 @@ module.exports = async (req, res, next) => {
     return next(error);
   }
   res.status(200).send(response);
-  return next();
```
Comment (Kajol-Kumari) [inline backend/app/routes/admin/getAdmins.js:49]: we should retain it

File: backend/app/routes/admin/updateAdmin.js [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
+      req.params.id,
+      { $set: updateFields },
+      { new: true }
+    );
+
         res.status(200).json(updatedAdmin);
       } catch (err) {
-        res.status(500).json(err);
-      }
-  res.send('Done');
```
Comment (Kajol-Kumari) [inline backend/app/routes/admin/updateAdmin.js:17]: we should retain this.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Screenshot 2024-05-19 at 11 57 23 AM

The update detail isn't getting reflected, atleast i should have seen the first name/lastname getting updated as image changes are in this pr

**Comment by Kajol-Kumari** [comment]
@SwayamRana808 let me know when u are done fixing the requested changes as i can see the current state of this pr is not good, a lot of un-necessary changes are there and functionality is still not intact

**Comment by SwayamRana808** [comment]
**i have changed response json but still it showing in network tab**

Screenshot 2024-05-20 212920

**it should have shown something like this --**
Screenshot 2024-05-20 213244

**@Kajol-Kumari  can you please have look on this**

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Screenshot 2024-05-22 at 9 40 52 PM
The phone number field keeps on giving error, can u check this?

**Comment by SwayamRana808** [comment]
> Screenshot 2024-05-22 at 9 40 52 PM The phone number field keeps on giving error, can u check this?

number should start with 9-6 it was validity check if number is from India its was already defined in backend

**Pull request merged**